qtconsole/console_widget.py

```diff
@@ -1722,9 +1716,17 @@ def _get_last_lines_from_list(self, text_list, num_lines):
     def _get_leading_spaces(self):
         """ Get the number of leading spaces of the current line.
         """
-        cur = self._get_cursor()
-        cur.select(QtGui.QTextCursor.LineUnderCursor)
-        text = cur.selectedText()[len(self._continuation_prompt):]
+
+        cursor = self._control.textCursor()
```

Comment:
`cursor = self._get_cursor()`

I know it's a bit silly, but we use `_get_cursor()` everywhere else. No good reason to change it here.

Reply:
Changed here and in the tests